<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordDialog</class>
 <widget class="QDialog" name="RecordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Record</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>241</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tabWeb">
    <attribute name="title">
     <string>Web</string>
    </attribute>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>351</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelWeb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Get from web sites.</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabText">
    <attribute name="title">
     <string>Text</string>
    </attribute>
    <widget class="QPlainTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>351</width>
       <height>151</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Paste a game record</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadTextButton">
     <property name="geometry">
      <rect>
       <x>259</x>
       <y>200</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load a game record in CSA or SFEN format.</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabUpload">
    <attribute name="title">
     <string>Load</string>
    </attribute>
    <widget class="QLabel" name="labelLoad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load a game record file of CSA or SFEN format.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fileOpenButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabDownload">
    <attribute name="title">
     <string>Save</string>
    </attribute>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save the current game record.</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
